conceptnet5/language/token_utils.py

```diff
@@ -1,15 +1,13 @@
 # coding: utf-8
-from __future__ import unicode_literals
 """
 This file contains some generally useful operations you would perform to
 separate and join tokens. The tools apply most to English, but should also
 be able to do their job in any Western language that uses spaces.
 """
+from __future__ import unicode_literals
```

Comment:
All the Python 2 hedging should be unnecessary; it's been quite a while since the ConceptNet codebase supported Python 2.